multirowgroup.go

```diff
@@ -0,0 +1,246 @@
+package parquet
+
+import (
+	"io"
+)
+
+// MultiRowGroup wraps multiple row groups to appear as if it was a single
+// RowGroup. RowGroups must have the same schema or it will error.
+func MultiRowGroup(rowGroups []RowGroup) (RowGroup, error) {
```

Comment:
To match the model of `io.MultiReader`, how about using an argument list instead?
```suggestion
func MultiRowGroup(rowGroups ...RowGroup) (RowGroup, error) {
```
I would also prefer not to return an error here and have the signature be
```suggestion
func MultiRowGroup(rowGroups ...RowGroup) RowGroup
```
Panicking on invalid input seems fine to me as it's an invalid use of the function.